### PR TITLE
Fixed moderate and critical a11y issues for realm settings

### LINKED ISSUE #17708: Accessibility/Realm setttings: Moderate Issues
### Describe the bug

The following moderate issues were found as per the [attached](https://docs.google.com/spreadsheets/d/1KtZJbDHKosSJBQm--1iJKQMwGIOtoohXIdfwMH3rPns/edit?usp=sharing).  

### Version

21.0.0

### Environment

_No response_

### Expected behavior

Reported issues are fixed.

### Actual behavior

Reported issues are not fixed.

### How to Reproduce?

Use Deque Browser Extension

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by agagancarczyk**
Closes https://github.com/keycloak/keycloak/issues/17708

### CODE REVIEW

**Review by edewit — DISMISSED** [review]

File: js/apps/admin-ui/src/realm-settings/realm-settings-section.css [hunk 1]
```diff
@@ -192,15 +192,15 @@ article.pf-c-card.pf-m-flat.kc-email-connection
 }
 
 .kc-emptyExecutors {
-  color: #8d9195;
+  color: #717477;
```
Comment (jonkoops) [inline js/apps/admin-ui/src/realm-settings/realm-settings-section.css:195]: Perhaps we can use some of the pre-defined PatternFly variables for colors here?

**Review by jonkoops — DISMISSED** [review]

File: js/apps/admin-ui/src/realm-settings/realm-settings-section.css (hunk 1 above)
Comment (agagancarczyk) [reply js/apps/admin-ui/src/realm-settings/realm-settings-section.css]: yeah, sure, great idea @jonkoops

**Review by jonkoops — APPROVED** [review]
LGTM

**Comment by jonkoops** [comment]
@agagancarczyk is there a reason to keep merging `main` in?

**Pull request merged**